doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
The plain text highlighting gets screwed up by it. Isn't going to look right to me showing file names as if they're prose, but if that's the more consistent direction to take things in, we should make that decision and write it down in the documentation standards since it's going to get inconsistent very quickly if people don't consciously think about this.